src/main/java/org/dataone/hashstore/filehashstore/FileHashStore.java

```diff
@@ -1458,6 +1670,21 @@ private void isStringEmpty(String string, String argument, String method) {
         }
     }
 
+    /**
+     * Checks whether a given long object is greater than 0
+     *
+     * @param object Object to check
+     * @param method Calling method
+     */
+    private void isObjectLessThanZero(long object, String method) {
```

Comment:
private void ~~isObjectLessThanZero~~ checkNotNegative(long longInt, String method) {

Reply:
Thank you for the suggestion - method name changed.